### PR TITLE
Fixes #5168 -  Localize disconnect alert message

### LINKED ISSUE #5168: [FxA] Localize message shown in alert on hit of disconnect
On settings page , when user hit disconnect we display alert with string "Are you sure you want to disconnect this device from your Firefox account?" We should review and localize this message.

### PR DESCRIPTION
**Pull request by punamdahiya**

### CODE REVIEW

File: locales/en-US/server.ftl [hunk 1]
```diff
@@ -291,6 +291,7 @@ settingsDescription = You can sign in with Firefox Accounts to sync all your scr
 settingsPageSubHeader = Sync & Accounts
 settingsClosePreferences =
     .title = Close preferences
+settingsFxaDisconnectAlertMessage = Are you sure you want to disconnect this device from your Firefox account?
```
Comment (6a68) [inline locales/en-US/server.ftl]: I think this should be 'Firefox Account', with the 'a' capitalized

**Comment by 6a68** [comment]
Marking as 'hold' until we get product feedback on the exact string, cc @johngruen

**Comment by johngruen** [comment]
String should be "Are you sure you want to disconnect this device from your Firefox Account?" with a capital A

**Comment by punamdahiya** [comment]
@flodolo Including to review localization strings in the PR. Thanks!

**Pull request merged**